lib/src/arbiter.dart

```diff
@@ -60,3 +60,57 @@ class PriorityArbiter extends Arbiter {
     ]);
   }
 }
+
+/// Arbiter Roun Robin.
+class RoundRobinArbiterlogic extends Arbiter {
+  ///Mask to define pending requests to be attended
+  ///i.e. If bit 4 is granted in a 8 bit request, [requestMask] will be 11110000
+  ///turning off the bit granted along with all previous bits
+  List<Logic> requestMask = [];
+
+  ///Result of masking [_requests] with [requestMask]
+  ///Contains bits of [_requests] but as bits granted, will off
+  List<Logic> grantMask = [];
+
+  /// Initiliazing Round Robin
```

Comment:
Let's elaborate a little more on the constructor, at least mentioning that it arbitrates between `requests`.